lib/structuraid_core/design_codes/aci_318_19/rc/footings/min_height.rb

```diff
@@ -0,0 +1,27 @@
+module StructuraidCore
+  module DesignCodes
+    module ACI31819
+      module RC
+        module Footings
+          class MinHeight
+            include DesignCodes::Utils::CodeRequirement
+            use_schema DesignCodes::Schemas::RC::Footings::MinHeightSchema
+
+            MIN_HEIGHT = 150
+            CODE_REFERENCE = 'ACI 318-19 13.3.1.2'.freeze
+
+            def call
+              return true if bottom_rebar_effective_height >= MIN_HEIGHT
+
+              raise RequirementNotFulfilledError.new(
+                :min_height,
+                "Effective Height #{bottom_rebar_effective_height} is below #{MIN_HEIGHT} mininmum",
+                CODE_REFERENCE
+              )
+            end
```

Comment:
I created this error to be used on any check that needs to be done by a code requirement. So, whenever we need to run this check but just know if it passed or not without breaking the program execution, we'll need to rescue it and apply the corresponding logic

Reply:
yeah, agree